server/database_setup.py

```diff
@@ -119,21 +121,28 @@
 }
 
 if __name__ == "__main__":
-    # print("Resetting database")
+    print("Resetting database")
     # base.Base.metadata.reflect(base.engine)
     # base.Base.metadata.drop_all(base.engine)
     # base.Base.metadata.create_all(base.engine)
-    # redis_connection.flushall()
+    redis_connection.flushall()
 
     item_types = {}
 
     print("Adding item types to database")
     with open(os.path.join(dirname, "app/database/data/item_types.json"), "r") as file:
         data = json.load(file)
         for record in data:
-            item_type = ModelItemType(name=record["en-US"])
+            item_type = ModelItemType()
+            for locale in record:
+                translation = ModelItemTypeTranslation(
+                    item_type_id=item_type.uuid, locale=locale, name=record[locale]
+                )
+                db.session.add(translation)
+                item_type.item_type_translation.append(translation)

```

Comment:
Are you sure you need this line? I think `item_type_id=item_type.uuid` may have this covered for you.